### PR TITLE
Release 2.2.0

### PR DESCRIPTION
**Pull request by hagenw**
image

For reference, `audeer.script_dir()` was added in https://github.com/audeering/audeer/pull/155

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -7,6 +7,16 @@ The format is based on `Keep a Changelog`_,
 and this project adheres to `Semantic Versioning`_.
 
 
+Version 2.2.0 (2024-07-23)
+--------------------------
+
+* Added: ``audeer.script_dir()``
+  which returns the directory of the file,
+  in which the function is called,
```
Comment (ChristianGeng) [inline CHANGELOG.rst:15]: I would say "  returns the directory of the file where it is called" or " returns the directory of the file from where it is called". I mean the "in which" that sounds strange to me.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -7,6 +7,16 @@ The format is based on `Keep a Changelog`_,
...
+  or the current working directory
+  when called in an interactive session
```
Comment (ChristianGeng) [inline CHANGELOG.rst:17]: Having no full stop at the end of a sentence is more common when it comes to CHANGELOG entries.

**Review by ChristianGeng — APPROVED** [review]
There are only cosmetic changes necessary. 

Approving straight away.

File: CHANGELOG.rst (hunk 1 above)
Comment (hagenw) [reply CHANGELOG.rst]: Good idea, I was also not happy with having two times "which". I changed to your first suggestion.

**Pull request merged**